prov/efa/src/rxr/rxr_ep.c

```diff
@@ -709,8 +709,14 @@ static void rxr_ep_free_res(struct rxr_ep *rxr_ep)
 	if (rxr_ep->readrsp_tx_entry_pool)
 		ofi_bufpool_destroy(rxr_ep->readrsp_tx_entry_pool);
 
-	if (rxr_ep->rx_readcopy_pkt_pool)
+	if (rxr_ep->rx_readcopy_pkt_pool) {
+		FI_WARN(&rxr_prov, FI_LOG_EP_CTRL, "current usage of read copy packet pool is %d\n",
```

Comment:
I'd change these to FI_INFO or FI_DBG not warn

Reply:
changed to FI_INFO in new revision.